examples/examples.py

```diff
@@ -92,6 +92,21 @@
 ipd.Audio(vca_out, rate=vca.sample_rate)
 # -
 
+# Alternately, you can just use the Drum class that composes all these modules together automatically.
 
+# +
+my_drum = Drum()
+
+boom = my_drum()
```

Comment:
let's just call this signal or X ? (and the STFT is Z?)

Reply:
ok. changed it to "drum_out" to match the formatting so far.